.github/workflows/generate-speakeasy-on-pr.yaml

```diff
@@ -54,12 +54,20 @@ jobs:
           mkdir -p ~/.speakeasy
           echo 'speakeasy_api_key: ${{ secrets.SPEAKEASY_API_KEY }}' > ~/.speakeasy/config.yaml
 
+      - name: Generate GitHub app token
+        id: github-app-token
+        uses: actions/create-github-app-token@67018539274d69449ef7c02e8e71183d1719ab42 # v2.1.4
+        with:
+          app-id: ${{ vars.KONG_GH_APP_ID }}
+          private-key: ${{ secrets.KONG_GH_APP_PRIVATE_KEY }}
```

Comment:
@mheap or @harshadixit12 could you please check if these two exist in this repo?

Reply:
These values should come from org secrets shouldn't they?

Reply:
I'm not sure. If they should then can I just merge this and check this works?